src/data_type/age.verification.js

```diff
@@ -1,24 +1,30 @@
 'use strict';
 
 const Validation = require('../yoti_common/validation');
+const constants = require('../yoti_common/constants');
 
 /**
- * Wraps an 'Age Verify/Condition' attribute to provide behaviour specific
+ * Based on an 'Age Verify/Condition' attribute name and value, provides behaviour specific
  * to verifying someone's age.
  *
  * @class AgeVerification
  */
 class AgeVerification {
-  constructor(attribute) {
-    Validation.notNull(attribute, 'attribute');
-    Validation.matchesPattern(attribute.getName(), /^[^:]+:(?!.*:)[0-9]+$/, 'attribute.name');
-    this.attribute = attribute;
+  static isAttributeNameMatchingAgeVerification(name) {
+    return name.startsWith(constants.ATTR_AGE_OVER)
+      || name.startsWith(constants.ATTR_AGE_UNDER);
+  }
+
+  constructor(name, value) {
+    Validation.isString(name, 'name');
+    Validation.oneOf(value, ['true', 'false'], 'value');
```

Comment:
Can we not use `Validation.isBoolean`?

Reply:
The actual value of the attribute is a string... agreed this is not the best, but that is what we received for it in the content type.

Reply:
Okay interesting, seems fine then if that is what we get from the API. Although we could coax those values similarly to how we are doing with dates. Though given it is of string type, it would be slightly more complex to achieve that. Ie we would have to check if the string is either `"true" || "false"` which may be a little bit magic. Anyway, don't want to introduce too much complexity here, so I think its probably fine to keep these string variants for now.

Reply:
Yeah, the main thing to solve is how we consume and transform:
- from a proto attribute, we should convert to what this content type is saying.
- from the converted attribute to its type, we could then convert to more business logic related type, and indeed coerced such fake boolean.

To be reviewed one day.